buildSrc/src/main/kotlin/Publishing.kt

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2016-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+@file:Suppress("UnstableApiUsage")
+
+import org.gradle.api.*
+import org.gradle.api.artifacts.dsl.*
+import org.gradle.api.provider.*
+import org.gradle.api.publish.maven.*
+import org.gradle.plugins.signing.*
+import java.net.*
+
+// Pom configuration
+
+infix fun <T> Property<T>.by(value: T) {
+    set(value)
+}
+
+fun MavenPom.configureMavenCentralMetadata(project: Project) {
+    name by project.name
+    description by "AtomicFU utilities"
+    url by "https://github.com/Kotlin/kotlinx.atomicfu"
+
+    licenses {
+        license {
+            name by "The Apache Software License, Version 2.0"
+            url by "https://www.apache.org/licenses/LICENSE-2.0.txt"
+            distribution by "repo"
```

Comment:
So nice! It makes me really wonder if Kotlin language shall support some kind of C++-style `operator fun assign` to overload an assignment of read-only properties (like `name`, `url`, etc), so that it can be written simply as:

```
name = "The Apache Software License, Version 2.0"
url = "https://www.apache.org/licenses/LICENSE-2.0.txt"
distribution = "repo"
```

The downside, is, of course, is that accessing `name`, `url`, `distribution` does not give you their values, but just a `Property` reference.